### DISCUSSION TITLE
<title>does github show my irl name when i file a dmca takedown

### DISCUSSION THREAD

**Discussion by AquaNot** [community/community#59990] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

so i was really curious and i was wondering if it shows my real-life name when i file a dmca takedown i dont know if it shows it or not can someone tell me?

**Comment by 3urobeat** [comment, marked as answer]
Based on all previous DMCA reports [listed here](https://github.com/github/dmca) your name will be hidden with `[private]`.

**Comment by tyzeron** [comment]
As stated, your private information is redacted from the [public DMCA repository](https://github.com/github/dmca).

However, in their [DMCA Takedown Policy](https://docs.github.com/en/site-policy/content-removal-policies/dmca-takedown-policy#e-transparency) they also mention the following:

> Please also note that, although we will not publicly publish unredacted notices, we may provide a complete unredacted copy of any notices we receive directly to any party whose rights would be affected by it.
